libs/lexical_cast/include/hpx/lexical_cast/detail/converter_numeric.hpp

```diff
@@ -0,0 +1,179 @@
+// Copyright Kevlin Henney, 2000-2005.
+// Copyright Alexander Nasonov, 2006-2010.
+// Copyright Antony Polukhin, 2011-2019.
+// Copyright Agustin Berge, 2019.
+//
+// SPDX-License-Identifier: BSL-1.0
+// Distributed under the Boost Software License, Version 1.0. (See
+// accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+// what:  lexical_cast custom keyword cast
+// who:   contributed by Kevlin Henney,
+//        enhanced with contributions from Terje Slettebo,
+//        with additional fixes and suggestions from Gennaro Prota,
+//        Beman Dawes, Dave Abrahams, Daryle Walker, Peter Dimov,
+//        Alexander Nasonov, Antony Polukhin, Justin Viiret, Michael Hofmann,
+//        Cheng Yang, Matthew Bradbury, David W. Birdsall, Pavel Korzh and other Boosters
+// when:  November 2000, March 2003, June 2005, June 2006, March 2011 - 2016
+
+#ifndef HPX_LEXICAL_CAST_DETAIL_CONVERTER_NUMERIC_HPP
+#define HPX_LEXICAL_CAST_DETAIL_CONVERTER_NUMERIC_HPP
+
+#include <hpx/config.hpp>
+
+#include <limits>
+#include <type_traits>
+
+#include <boost/numeric/conversion/cast.hpp>
```

Comment:
Can this be forked as well?

Reply:
This is an entire module, and should be handled separately.

Reply:
This is needed for this test only... Should we remove the test?

Reply:
Sounds good to me

Reply:
...which test is that?

Reply:
Uhh, my bad - it's not a test.